vscode-plugin/README.md

```diff
@@ -0,0 +1,10 @@
+## Ballerina Integrator VSCode Plugin
+
+#### Adding new snippets to the plugin
+
+1. Create the relevant code snippet by following the format given in the file, SnippetGenerator.java.
```

Comment:
Shall we provide the relative path of the `SnippetGenerator.java`?

Reply:
Fixed